pages/home/oracle-integrity-staking/examples.mdx

```diff
@@ -0,0 +1,172 @@
+import StakingCapBar from "@/components/StakingCapBar";
+import RewardSimulator from "@/components/RewardSimulator";
+
+# Examples
+
+This reference page provides examples of various scenarios to illustrate the Mathematical Representations of OIS.
+
+NOTE: All the symbols used in the examples are explained in the [Mathematical Representation](/home/pyth-token/oracle-integrity-staking/mathematical-representation) section.
+
+## Example 1: Only Publisher Stake
+
+This example take the case one pool where the pool has stake from only the publisher.
+
+$$
+\begin{aligned}
+{S^p_p} &= 100 \\
+{S^d_p} &= 0 \\
+{S^p} &= {S^p_p} + {S^d_p} = 100 + 0 = 100 \\
+{C}_p &= 500 \\
+\text{Total Amount eligible for Reward} \quad{E_p} &= min({S}_p, {C}_p) = min(500, 100) = 100 \\
+\text{Reward Rate (Yearly)} \quad{r} &= 10\% \\
+
+\text{Total Reward for one year} \quad{R_p} &= {r} \times {E_p} = 10\% \times 100 = 10 \\
+\text{Publisher Reward} \quad{R^p_p} &= {r} \times min({S^p_p}, {C}_p) = 10\% \times 100 = 10 \\
+\text{Delegator Reward} \quad{R^d_p} &= {R_p} - {R^p_p} = 10 - 10 = 0 \\
```

Comment:
perhaps you can add effective yield rate of 
publisher and delegator

effective yield rate is equal to
R^{p}_{p} / S^{p}_{p} for publisher and
R^{d}_{p} / S^{d}_{p} for delegator